app/__init__.py

```diff
@@ -77,7 +77,7 @@ def campsite_manipulation(id, **kwargs):
             'lon': campsite.lon,
             'lat': campsite.lat,
 						'timestamp': campsite.date_created,
-						'amenities': str(campsite.list_amenities())
+						'amenities': campsite.list_amenities()
```

Comment:
Amenities were forced into a string here. Now a collection is sent through.

Reply:
Looks great, thanks!